### PR TITLE
Server api

### PR DESCRIPTION
**Pull request by AbnerLin**
Please review and approve, thx.

### CODE REVIEW

File: daemon/database/artistrepo.py
Comment (rockers7414) [inline daemon/database/artistrepo.py]: This function should be named `get_artist_by_name` since we may have other ways to get the artist, such as `source id`.

File: daemon/decorator/serialize.py
Comment (rockers7414) [inline daemon/decorator/serialize.py]: The plural of entity is entities.

File: daemon/database/albumrepo.py
Comment (rockers7414) [inline daemon/database/albumrepo.py]: This function should be named `get_album_by_name`.

**Review by rockers7414 — CHANGES_REQUESTED** [review]
Please revise per request.

**Comment by rockers7414** [comment]
Once your PR is merged, I'll cover @KocaMio to finish the client side.

**Comment by AbnerLin** [comment]
What is PR?

**Comment by rockers7414** [comment]
PR stands for pull request.

**Review by rockers7414 — APPROVED** [review]
Looks good. I'll merge it tonight if no one has any concern.

**Pull request merged**
